core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info.model.ts

```diff
@@ -97,10 +139,18 @@ export class EntityEditorBrowserTabsInfo {
     return this._numberOfOpenedTabs;
   }
 
+  /**
+   * Increments the number of opened tabs by one. It is required
```

Comment:
Not sure what "it" refers to here. Could you please rephrase? (Or actually -- I don't think you even need the second sentence, but if you want to keep it I guess you can write "It should be called when ...".)

Ditto below.

Reply:
Removed the second sentences now.